includes/class-wc-payments-token-service.php

```diff
@@ -17,6 +17,8 @@
  */
 class WC_Payments_Token_Service {
 
+	const CUSTOMER_ID_META_KEY = '_wcpay_customer_id';
```

Comment:
It'll be good to also call this `DEPRECATED_WCPAY_CUSTOMER_ID_OPTION` as it is in the other PR.

Reply:
This specifies the meta key in which the customer ID is going to be attached to the token. It's the new default behavior to attach the customer to the token to avoid displaying tokens that will not be able to be used during checkout. Did you mean that we should name it `WCPAY_CUSTOMER_ID_OPTION`?

If so, I'd rather keep it as `META_KEY` due to the way it's used. The `CUSTOMER_ID_OPTION` ones are used to deal with user options using the `(get|update|delete)_user_option` functions, while this one is used to handle the token's metadata using the functions `(add|update)_meta_data`, `get_meta ` and `meta_exists`. Does that make sense?

Reply:
Cleared out in Slack, the name made sense pasting Luiz's explanation here for future comment readers:


`_wcpay_customer_id` in the Token_Service relates to the wcpay_customer_id for that token, while `_wcpay_customer_id_live` and `_wcpay_customer_id_test` relate to the `wcpay_customer_id` attached to the WooCommerce user

The `wcpay_customer_id` is our Stripe customer entity. It has a relationship with the WC_User in the sense that we create Stripe customers for the WC_Users, but it also has a relationship with the token, as a Stripe user owns a Payment Method, which is stored in the token to enable saved cards and subscriptions.

Reply:
I've added some comments to clarify that as well in https://github.com/Automattic/woocommerce-payments/pull/972/commits/d26d052f13925083fceb179bb012611748822b9e and https://github.com/Automattic/woocommerce-payments/pull/973/commits/aa26d7959aabd9b77583bf9a2765723dc2157d03.